app/src/main/java/com/hedvig/app/util/featureflags/FeatureManager.kt

```diff
@@ -1,29 +1,27 @@
 package com.hedvig.app.util.featureflags
 
 import androidx.annotation.VisibleForTesting
-import com.hedvig.app.feature.settings.MarketManager
+import com.hedvig.app.feature.settings.Market
+import com.hedvig.app.isDebug
 import java.util.concurrent.CopyOnWriteArrayList
 
-class FeatureManager(
```

Comment:
I had to remove marketManager injection here because of circular dependencies. Passing it to the function instead.